packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -290,3 +290,30 @@ When('I see {string} as the {string} input', (custom, order) => {
 
   // cy.findByLabelText(custom).type(Cypress.env('VALID_PASSWORD'));
 });
+
+When('I type a valid code', () => {
+  /**
+   * Confirmation code differs on React/Vue vs Angular. Testing for both for
+   * now, but we can look to make them consistent on next major release.
+   */
+  const regex = new RegExp(`^(confirmation )?code( *)?`, 'i');
+  cy.findByRole('spinbutton', { name: regex }).type('1234');
```

Comment:
Moved these to common so they could be accessed by more then the reset password feature